libs/go/libp2p_node/dht/dhtpeer/dhtpeer_test.go

```diff
@@ -758,6 +758,67 @@ func TestRoutingDelegateClientToDHTPeerIndirect(t *testing.T) {
 	expectEnvelope(t, client.Rx)
 }
 
+// TestMessageOrderingWithDelegateClient
+func TestMessageOrderingWithDelegateClient(t *testing.T) {
```

Comment:
This doesn't test the message ordering at reception

Reply:
It does now, have a look